recipe-client-addon/bin/tc/test.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -eu
+
+# mach wants this
+export SHELL=$(which bash)
+
+apt-get install -y npm curl python2.7 xvfb
+
+# Creates gecko-dev-master
+echo 'Downloading gecko-dev...'
+curl -sL https://github.com/mozilla/gecko-dev/archive/master.tar.gz | tar xz
```

Comment:
I actually started by using taskcluster-vcs, and was steered away from it by garndt. Apparently it isn't well maintained, and is probably going away in the future.

There may be some improvements possible here, but I think there will be much better payoff in improving our total build times.